shipyard-controller/internal/handler/debughandler_test.go

```diff
@@ -459,3 +459,130 @@ func TestDebughandlerGetAllEvents(t *testing.T) {
 		require.Equal(t, tt.shkeptncontext, tt.fields.DebugManager.GetAllEventsCalls()[0].Shkeptncontext)
 	}
 }
+
+func TestDebughandlerGetBlockingSequences(t *testing.T) {
+
+	type fields struct {
+		DebugManager *fake.IDebugManagerMock
+	}
+
+	sequences := []models.SequenceExecution{
+		{
+			ID: "my-id",
+			Sequence: keptnv2.Sequence{
+				Name: "delivery",
+			},
+			Status: models.SequenceExecutionStatus{
+				State: apimodels.SequenceTriggeredState,
+			},
+			Scope: models.EventScope{
+				EventData: keptnv2.EventData{
+					Project: "my-project",
+					Stage:   "my-stage",
+					Service: "my-service",
+				},
+				KeptnContext: "my-context-id5",
+			},
+		},
+	}
+
+	tests := []struct {
+		name           string
+		fields         fields
+		request        *http.Request
+		wantResponse   []models.SequenceExecution
+		wantStatus     int
+		projectName    string
+		shkeptncontext string
+		stage          string
+	}{
```

Comment:
I would add a test to check if a stage doesn't exist there are no sequences returned and there is no error.